### ISSUE TITLE
Missing node

### ISSUE THREAD

**Issue by MaxZimmer** [ncguilbeault/BonZeb#2]
Hello, 

Thanks for sharing this work.
I am currently playing with BonZeb and I think you may have a label issue in Examples/Behavioural Tracking and Analysis/
Any chance you may have replaced CalculateTailCurvature with CalculateTailAngle ?

**Comment by ncguilbeault** [comment]
Yes! I updated the name of the node in the github repository but haven't pushed it to the Bonsai manager yet... Still working on it :) The 2 nodes are basically interchangeable. The version from the Bonsai manager should still be the old one, which will work with all of the workflows
